lib/conseil_lib/convert_to_micheline.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+exports.__esModule = true;
+var conseiljs_1 = require("conseiljs");
+if (process.argv.length != 3) {
+    console.log("USAGE: node convert_to_moicheline.js CODE\nARGS:: \n\tCODE: michelson script");
```

Comment:
- Why do we send 3 args if we only use one of them?
["node", "file.js", "arg"].length == 3

- in case of error, can we show this on the error output and return a non 0 code, so that the caller is aware of the issue?
ok

- wouldn't michelson_to_micheline.js be a more precise name for the file?
you prefer `convert_michelson_to_micheline.js`?

Reply:
> * Why do we send 3 args if we only use one of them?
>   ["node", "file.js", "arg"].length == 3
ha, ok.

> * wouldn't michelson_to_micheline.js be a more precise name for the file?
>   you prefer `convert_michelson_to_micheline.js`?
perfect!